docs/content/users/install/docker-installation.md

```diff
@@ -46,8 +46,9 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     ### Docker Desktop for Mac
 
     Docker Desktop for Mac can be installed via Homebrew (`brew install homebrew/cask/docker`) or can be downloaded from [docker.com](https://www.docker.com/products/docker-desktop). It has long been supported by DDEV and has extensive automated testing.
-
-    We do not recommend the `VirtioFS` option with Docker Desktop for Mac. While it’s performant, it can cause mysterious problems that are not present with [Mutagen](performance.md#mutagen)—which offers comparable performance when enabled.
+    
+    !!!Warning
```

Comment:
I’d lowercase the `w` for consistency with the rest of the docs, and consider adding a title to summarize the blurb for easier scanning:

```suggestion
    !!!warning "Ports unavailable?"
```